hub/powertoys/run.md

```diff
@@ -241,8 +241,8 @@ Examples:
 
 ### Unit converter plugin
 
-> [!NOTE]
-> The Unit Converter plugin respects the number format settings of your system. Please be aware of the different decimal and thousand delimiters in different locals. The names and abbreviations of the units aren't localized yet.
+> [!IMPORTANT]
+> The Unit Converter plugin respects the number format settings of your system. Please be aware of the different decimals and thousand delimiters in different locals. The names and abbreviations of the units aren't localized yet.
```

Comment:
I think decimals should remain decimal. It's talking about "decimal delimiters" and "thousand delimiters".

Reply:
Yeah, subtle but tricky. "Decimal delimiters" doesn't sound right. There can be only one decimal char, so it can't be delimited. Another word could be "_thousand(s)_ separator". Might be best to split this into "decimal character and thousands-separator".

Reply:
Yes, I like your suggestion to break it up.